src/content/api/asset-transfers.mdx

```diff
@@ -115,6 +116,62 @@ Asset Transfer goes through different lifecycle stages.
 
 ---
 
+<Endpoint
+  path="/api/asset-transfer-requests"
+  filename="asset-transfers-request"
+>
+  ## Request an Asset Transfer
+  ### Experimental API
+
+
+  Request a transfer of an asset from a sender to the recipient.
+
+  <Properties>
+    <Property name="recipientAlias" type="string" required>
+      Phone number, email or handle of the person who request for asset transfer.
+    </Property>
+
+    <Property name="senderAccountId" type="string" required>
+      The id of the sender who receive the request.
+    </Property>
+
+    <Property name="assetType" type="string" required>
+      The type of asset being transferred. Currently, supports only `quartz.nzd` assets
+    </Property>
+
+    <Property name="description" type="string">
+      Shows up in transaction history against the transfer. 200 character limit.
+    </Property>
+
+    <Property name="message" type="string">
+      Shows up in transaction history against the transfer. 100 character limit.
+    </Property>
+
+    <Property name="value" type="bignumber">
+      Amount to send. Required for money transfers. Units depend on the asset type.
```

Comment:
This value is not required in any case currently as it can be defined in the accept endpoint